src/data/option/basic.lean

```diff
@@ -106,6 +106,18 @@ by cases x; simp
   x.map f = some b ↔ ∃ a, x = some a ∧ f a = b :=
 by cases x; simp
 
+lemma map_eq_none {α β} {x : option α} {f : α → β} :
```

Comment:
```suggestion
@[simp] lemma map_eq_none {α β} {x : option α} {f : α → β} :
```
like below? Or is there a reason not to?

Reply:
simp linter complains if both are `simp` afair.